### PR TITLE
Solves #37 by removing js links in the head.html

### PR DESCRIPTION
**Pull request by thechargedneutron**

### CODE REVIEW

File: _site/soc_projects/10-fossee-scilab-julia.html [hunk 1]
```diff
@@ -325,6 +319,23 @@ <h3 id="week-1">Week 1</h3>
...
+  <li>Added macOS Julia libraries and tried linking them</li>
+</ul>
+
+<h3 id="week-3">Week 3</h3>
+<ul>
+  <li>Added support for scalar types conversions for all three types</li>
+  <li>Added boolean support completed and tested</li>
+  <li>Added code for Hypermat doubles extraction from scilab but can’t inject into julia environment</li>
+  <li>Added similar code for Hypermat int and boolean conversion</li>
+</ul>
```
Comment (nihal111) [inline _site/soc_projects/10-fossee-scilab-julia.html]: Can I have this in a separate PR?

**Comment by thechargedneutron** [comment]
I have not added this progress, I found out that this was Rishika's commit
and was merged 7 days ago, how come this be in my PR, I am not sure  -_-

**Comment by nihal111** [comment]
Not your fault, it wasn't built yet. You just built it. So the change appeared in the _site/.. file

**Comment by TheFlash98** [comment]
@thechargedneutron you've been pushing all your commits to master hence the PR you send has so many commits. Although it works, try making a new branch every time you have to send a PR, push to that branch and once it's merged, delete the branch! 
Cheers! 😄

**Comment by nihal111** [comment]
This breaks a few things. The first thing I noticed was that the banner was resized and looked bad. The script for skel.min.js and skel-layers.min.js are important and they deal with positioning and sizing of elements I guess. See if you can fix this somehow,.

**Comment by thechargedneutron** [comment]
Okk, will let you know

**Comment by cheekujodhpur** [comment]
Yes, you cannot simply remove those links. You have to ensure those are
loaded and executed after all image elements are loaded, but NOT remove
them.

**Comment by cheekujodhpur** [comment]
I think the skel.js uses/needs jQuery no?

**Comment by cheekujodhpur** [comment]
I think we all agree this is not a good idea. Should we close this?

**Comment by TheFlash98** [comment]
@cheekujodhpur I'm gonna close this as well.

**Pull request closed without merging**